couchbase-persistence/couchbase-persistence-scala-sbt/build.sbt

```diff
@@ -39,12 +28,17 @@ lazy val `hello-impl` = (project in file("hello-impl"))
       //#couchbase-begin
       "com.lightbend.akka" %% "lagom-scaladsl-persistence-couchbase" % "1.0",
       //#couchbase-end
+
+      // lagom-scaladsl-persistence-couchbase doesn't depend on the latest versions for Lagom and Akka
+      // so explicit dependencies must be added to ensure versions align.
+      // Also, akka-persistence-typed is not a transitive dependency in lagom-scaladsl-persistence-couchbase
+      // so it must be added explicitly.
+      lagomScaladslPersistence, // align versions
+      "com.typesafe.akka" %% "akka-persistence-typed" % akkaVersion, // align versions and add it in scope
+
       lagomScaladslTestKit,
       macwire,
       scalaTest,
-      "com.typesafe.akka" %% "akka-testkit" % akkaVersion,
-      "com.typesafe.akka" %% "akka-stream-testkit" % akkaVersion,
-      "com.typesafe.akka" %% "akka-persistence-query" % akkaVersion,
```

Comment:
These are probably the changes that took me the most. I added the comments to explain what's going on.

Unlike Cassandra and JDBC, Lagom's support for Couchbase is stuck in Lagom `1.5` so there are several dependencies that must be brought in explicitly or versions that need overriding. E.g. unless overriden with `lagomScaladslPersistence`, the project will use the artifact brought in transitively by `lagom-scaladsl-persistence-couchbase` which is the wrong version.


This is the minimal override I could find that works:

```
      lagomScaladslPersistence, 
      "com.typesafe.akka" %% "akka-persistence-typed" % akkaVersion, 
```